packages/ui/src/theme/css/component/inAppMessaging/backdrop.scss

```diff
@@ -0,0 +1,11 @@
+.amplify-backdrop {
+  align-items: center;
```

Comment:
Might customers want to customize some of these other properties besides background-color? I don't have enough context to make a conclusion either way.

Reply:
We have no real intent/requirements/guidance one way or another. I felt this was a "bare minimal" starting point and future customizations would be additive

Reply:
IMO we shouldn't be offering more than a background color override here for the time being. The use case of this component is create a full screen overlay, and until we need more from it agree with the approach @cshfang has taken here